### PR TITLE
add missed changes from previous PR

### PR DESCRIPTION
**Pull request by FjorgeTheHuman**
because mugman and #13

### CODE REVIEW

**Comment by FjorgeTheHuman** [comment]
OMG FFS THERES A CONFLICT REEEEEEEEEEEEEE

File: libtwirl/__main__.py [hunk 1]
```diff
@@ -1,24 +1,25 @@
 from libtwirl.errors import IncorrectArgumentFormatError
 from libtwirl.core.install import pkg_install
+from libtwirl.core.remove import pkg_remove
 from sys import argv
 
 args = argv[1:]
+pkgs = args[1:]
```
Comment (mugman174) [inline libtwirl/__main__.py:7]: Maybe have a try except block if someone doesn't give an arg

Comment (FjorgeTheHuman) [reply libtwirl/__main__.py:7]: maybe I would've if u didn't merge the PR so soon kek

**Comment by mugman174** [comment]
> OMG FFS THERES A CONFLICT REEEEEEEEEEEEEE

It's a smol one tho, so not too bad.

File: libtwirl/__main__.py (hunk 1 above)
Comment (caughtquick) [reply libtwirl/__main__.py:7]: alr chill out, it was an accident and I shouldn't have approved it when an error existed

Comment (mugman174) [reply libtwirl/__main__.py:7]: sry I prolly misclicked idk ¯\_(ツ)_/¯

Comment (FjorgeTheHuman) [reply libtwirl/__main__.py]: fixed in latest version

Comment (caughtquick) [reply libtwirl/__main__.py]: looks good outside of that weird conflict, all errors seem to be resolved, checking one last time then approving

**Review by mugman174 — DISMISSED** [review]
My brain is sleeping a bit rn but I think it's ok now.

**Comment by caughtquick** [comment]
still says conflicts exist, can you resolve em so  I can approve?

**Comment by FjorgeTheHuman** [comment]
conflicts should be resolved:tm:

**Comment by caughtquick** [comment]
resolved, final check then approving

**Review by mugman174 — APPROVED** [review]
oke

**Comment by caughtquick** [comment]
lgtm, approving now

**Review by caughtquick — APPROVED** [review]

**Comment by mugman174** [comment]
It's ready...

**Comment by caughtquick** [comment]
squashing and merging cause its 15 commit and it won't let me do a merge commit either way

**Comment by caughtquick** [comment]
@thewarsawpakt please see if you can resolve the permissions issue making it so we can't squash and merge

**Pull request merged**